.gitignore

```diff
@@ -2,4 +2,5 @@
 **/__pycache__
 bert_models/*
 classifier_models/*
-tmp
\ No newline at end of file
+.credz
+.vscode
```

Comment:
The project already has checked-in .vscode/ stuff, so please don't gitignore them